nowcasting_utils/models/losses/TotalVariationLoss.py

```diff
@@ -0,0 +1,28 @@
+import torch
+from torch import nn as nn
+
+
+def tv_loss(img, tv_weight):
+    """
+    Taken from https://github.com/chongyangma/cs231n/blob/master/assignments/assignment3/style_transfer_pytorch.py
```

Comment:
https://github.com/chongyangma/cs231n lacks an explicit license.  Please ask chongyangma if they're happy for us to copy their code, and how they'd like to be attributed :)

(Sorry for being a pedant about licenses!)

Reply:
I have found an Apache Licensed version that also has some other nice features, so swapping it out for that!